tsconfig.json

```diff
@@ -30,8 +30,9 @@
         "overrideIdFn": "[sha512:contenthash:base64:6]",
         "ast": true
       }
-    ]
+    ],
+    "types": ["node", "jest", "@testing-library/jest-dom"],
   },
   "include": ["src"],
-  "exclude": ["node_modules", "lib"]
+  "exclude": ["node_modules", "lib"],
```

Comment:
Typescript supports comments and commas, but yes better stick with the standard if this is ever parsed by an external tool

Reply:
Yeah I've got some plugins crashing in my editor on config files with trailing comma's because they use the Python API which does not allow trailing comma's.

I'm okay with trailing comma's/comments if the appropriate extension is used (like https://json5.org/), but in this case best to just stick to plain JSON.